zap/src/main/java/org/zaproxy/zap/view/ContextIncludePanel.java

```diff
@@ -64,6 +75,56 @@ public static String getPanelName(Context context) {
         return getPanelName(context.getId());
     }
 
+    private File selectImportFile() {
+        JFileChooser chooser = new JFileChooser(Constant.getContextsDir());
+        File file = null;
+
+        int rc = chooser.showOpenDialog(View.getSingleton().getMainFrame());
+        if (rc == JFileChooser.APPROVE_OPTION) {
+            file = chooser.getSelectedFile();
+            if (file == null || !file.exists()) {
+                return null;
+            }
+            return file;
+        }
+        return null;
+    }
+
+    private void addImportButton() {
+        String IMPORT_BUTTON_LABEL = Constant.messages.getString("context.include.import.label");
+        JButton importButton = new JButton(IMPORT_BUTTON_LABEL);
+        importButton.addActionListener(
+                new ActionListener() {
+                    @Override
+                    public void actionPerformed(ActionEvent evt) {
+                        File file = selectImportFile();
+                        if (file != null) {
+                            List<String> regexes = regexesPanel.getRegexes();
+
+                            String line = null;
+                            try (BufferedReader br = new BufferedReader(new FileReader(file))) {
+                                while ((line = br.readLine()) != null) {
+                                    String trimmedRegex = line.trim();
+                                    if (!regexes.contains(trimmedRegex)) {
+                                        Pattern.compile(trimmedRegex, Pattern.CASE_INSENSITIVE);
```

Comment:
Hmmmm I'm not sure about assuming case insensitive. Users can always designate `(?i)` at the start of their pattern if they wish. Unless you've checked and this behavior is assumed elsewhere in context handling.

Reply:
This is the same behavior as for the add button (MultipleRegexesOptionsPanel.validateFields) and I did the same validation for consistency